dataactvalidator/config/sqlrules/sqlRules.csv

```diff
@@ -14,151 +14,151 @@ A11,SpendingAuthorityfromOffsettingCollectionsAmountTotal_CPE= CPE aggregate val
-FABS23,"AwardingSubTierAgencyCode must be a valid 4-digit sub-tier agency code.","AwardingSubTierAgencyCode must be a valid 4-digit sub-tier agency code.",n,detached_award,fatal,,fabs23_detached_award_financial_assistance
-FABS24,"PrimaryPlaceOfPerformanceCountryCode must contain a valid three character GENC Standard Edition 3.0 (Update 4) country code.","PrimaryPlaceOfPerformanceCountryCode must contain a valid three character GENC Standard Edition 3.0 (Update 4) country code.",n,detached_award,fatal,,fabs24_detached_award_financial_assistance
+FABS23,AwardingSubTierAgencyCode must be a valid 4-digit sub-tier agency code.,AwardingSubTierAgencyCode must be a valid 4-digit sub-tier agency code.,n,detached_award,fatal,,fabs23_detached_award_financial_assistance
+FABS24,PrimaryPlaceOfPerformanceCountryCode must contain a valid three character GENC Standard Edition 3.0 (Update 4) country code.,PrimaryPlaceOfPerformanceCountryCode must contain a valid three character GENC Standard Edition 3.0 (Update 4) country code.,n,detached_award,fatal,,fabs24_detached_award_financial_assistance
 FABS25,"AwardDescription is required for non-aggregate records (i.e., when RecordType = 2).","AwardDescription is required for non-aggregate records (i.e., when RecordType = 2).",n,detached_award,fatal,,fabs25_detached_award_financial_assistance
-FABS26,"FederalActionObligation must be blank or 0 for loans (i.e., when AssistanceType = 07 or 08).","FederalActionObligation must be blank for loans (i.e., when AssistanceType = 07 or 08).",n,detached_award,fatal,,fabs26_detached_award_financial_assistance_1
-FABS26,"FederalActionObligation is required for non-loans (i.e., when AssistanceType is not 07 or 08).","FederalActionObligation is required for non-loans (i.e., when AssistanceType is not 07 or 08).",n,detached_award,fatal,,fabs26_detached_award_financial_assistance_2
-FABS27,"NonFederalFundingAmount must be blank or 0 for loans (i.e., when AssistanceType = 07 or 08).","NonFederalFundingAmount must be blank for loans (i.e., when AssistanceType = 07 or 08).",n,detached_award,fatal,,fabs27_detached_award_financial_assistance
-FABS28,"FaceValueLoanGuarantee is required for loans (i.e., when AssistanceType = 07 or 08).","FaceValueLoanGuarantee is required for loans (i.e., when AssistanceType = 07 or 08).",n,detached_award,fatal,,fabs28_detached_award_financial_assistance_1
-FABS28,"FaceValueLoanGuarantee must be blank or 0 for non-loans (i.e., when AssistanceType is not 07 or 08).","FaceValueLoanGuarantee must be blank or 0 for non-loans (i.e., when AssistanceType is not 07 or 08).",n,detached_award,fatal,,fabs28_detached_award_financial_assistance_2
-FABS29,"OriginalLoanSubsidyCost is required for loans (i.e., when AssistanceType = 07 or 08).","Orig
```

Comment:
Can you check with Ross to make sure the extra "or" was intended? It seems very odd.

Reply:
I dont understand. where is this?